main.css

```diff
@@ -71,7 +92,7 @@
   height: 100%;
   width: 0;
   position: fixed;
-  z-index: 1;
```

Comment:
This is great! But theres 2 z-index's in places which will override each other. Should I still merge?